test/test_ops.py

```diff
@@ -423,6 +464,49 @@ def test_reference_eager(self, device, dtype, op):
     for sample_input in sample_inputs:
       self.compare_with_eager_reference(op, sample_input)
 
+  @ops(fallback_ops_to_test, allowed_dtypes=(torch.float32, torch.long))
+  def test_fallback(self, device, dtype, op):
+    if self.device_type != 'xla':
+      self.skipTest("This test runs only on XLA")
+
+    fallback_info = allowed_fallback_opinfo[get_name(op)]
+
+    # Maximum number of samples to run.
+    # Running all of them might take too much time.
+    MAX_SAMPLES_TO_RUN = 2
+
+    # Expected fallback operation names.
+    expected_fallbacks = fallback_info.fallback_ops
+
+    # Filter so that only allowed samples are run.
+    sample_inputs = [
+        sample for sample in op.sample_inputs(
+            device, dtype, requires_grad=dtype.is_floating_point) if
+        fallback_info.allow_sample is None or fallback_info.allow_sample(sample)
+    ]
+    samples_to_run = min(MAX_SAMPLES_TO_RUN, len(sample_inputs))
+
+    # Test CPU and CUDA fallbacks.
+    for xla_fallback_cuda in ("0", "1"):
+      os.environ[xenv.XLA_FALLBACK_CUDA] = xla_fallback_cuda
+
+      with self.subTest(XLA_FALLBACK_CUDA=xla_fallback_cuda):
+        for i in range(samples_to_run):
+          sample_input = sample_inputs[i]
+
+          # Clear all metrics.
+          # This should also clear the fallbacks.
+          xmetrics.clear_all()
```

Comment:
nit: might be better to be consistent with other files `import torch_xla.debug.metrics as met` and `met.clear_all()`